src/document-click.js

```diff
@@ -108,6 +113,21 @@ export function onDocumentElementClick_(e, viewport) {
   const win = doc.defaultView;
 
   const tgtLoc = parseUrl(target.href);
+
+  // On Safari iOS, custom protocol links will fail to open apps when the
+  // document is iframed - in order to go around this, we set the top.location
+  // to the custom protocol href.
+  const isSafariIOS = platform.isIos() && platform.isSafari();
+  const isEmbedded = win.parent && win.parent != win;
+  const isCustomProtocolHref = (
+      tgtLoc.protocol != 'http:' && tgtLoc.protocol != 'https:');
+  if (isSafariIOS && isEmbedded && isCustomProtocolHref) {
+    win.top.location.href = target.href;
```

Comment:
Did you try these options?
```
window.open(URL, '_blank');
```
or
```
window.open(URL, '_top');
```

This is the order of my preference if any of these work.